README.md

```diff
@@ -40,8 +40,6 @@ gem 'scimitar', '~> 2.0'
 
 Scimitar uses [semantic versioning](https://semver.org) so you can be confident that patch and minor version updates for features, bug fixes and/or security patches will not break your application.
 
-
-
```

Comment:
FYI those were deliberate as it's much easier to skim through the source code of Markdown and see where major headings are if there's a visual break. The file ought to be consistent with that style of line breaking before L2 headings (and L3 on a judgement call depending on context and surrounding information density).

This won't halt you PR but I'll be manually restoring that once it's merged in a subsequent commit.

Reply:
@pond I'm sorry, this was a markdown linter. I'll revert the undesired changes.